sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -1657,39 +1621,19 @@ private void updateFollowUpStatusComponents() {
 		}
 	}
 
-	private Map<ReinfectionDetail, Boolean> mergeReinfectionTrees() {
-		Map<ReinfectionDetail, Boolean> reinfectionDetails = new EnumMap<>(ReinfectionDetail.class);
-		reinfectionTrees.values().forEach(t -> {
-			if (t.getValues() != null) {
-				reinfectionDetails.putAll(t.getValues());
-			}
-		});
-		return reinfectionDetails;
-	}
-
 	@Override
 	public CaseDataDto getValue() {
```

Comment:
please delete these method becaus there is no more need to set the reinfection field separately

Reply:
removed